packages/component/src/Activity/private/FeedbackForm.tsx

```diff
@@ -23,14 +24,24 @@ function FeedbackForm({ feedbackType, disclaimer, onReset, replyToId }: Feedback
   const localize = useLocalizer();
   const onResetRef = useRefFrom(onReset);
   const postActivity = usePostActivity();
+  const renderMarkdownAsHTML = useRenderMarkdownAsHTML('message activity');
 
-  const handleReset = useCallback(() => {
-    setUserFeedback('');
+  const disclaimerHtml = {
+    __html: disclaimer ? renderMarkdownAsHTML(disclaimer) : undefined
+  };
```

Comment:
`renderMarkdownAsHTML` will be `undefined` on minimal bundle or if C1 explicitly removed it. Sorry our TypeScript config isn't as strict as it should yet, right now, it hide `undefined`.

```suggestion
  const disclaimerNode = useMemo(
    () =>
      disclaimer ? (
        renderMarkdownAsHTML ? (
          <span
            className={classNames('webchat__feedback-form__caption1', feedbackForm + '')}
            // eslint-disable-next-line react/no-danger
            dangerouslySetInnerHTML={{ __html: renderMarkdownAsHTML(disclaimer) }}
          />
        ) : (
          <span className={classNames('webchat__feedback-form__caption1', feedbackForm + '')}>{disclaimer}</span>
        )
      ) : undefined,
    [disclaimer, feedbackForm, renderMarkdownAsHTML]
  );
```

Reply:
You can also consider refactor it into a component so it looks better. Your call.